review_july (#172)
fiskaltrust/productdescription-de-doc: glossar/README.md

### Kassenseriennummer

> Die Kassenseriennummer ist eine **eindeutige** Information, die auf den Beleg gedruckt werden muss. Dazu wird von fiskaltrust die ftCashBoxIdentification verwendet, Base64-kodiert, Sonderzeichen entfernt  und von der fiskaltrust.Middleware im Signatur-lock in der Response (ftSignatureType) zurückgeliefert. Die ftCashBoxIdentification selbst ist die frei wählbare, **eindeutige** Bezeichnung für eine Queue. Sie wird auch von der SCU als ClientId für die TSE verwendet. Deshalb ist es wichtig hierbei einen "printable string" mit maximal 20 Zeichen einzugeben. 

Reviewer: Nach der KassenSichV ist die "Seriennummer des elektronischen Aufzeichnungssystems oder die Seriennummer des Sicherheitsmoduls". 
Eine Kassenseriennummer kennt weder KassenSichV noch DSFinV-K.

Reviewer: ```suggestion
Die Kassenseriennummer ist eine **eindeutige** Information, die auf den Beleg gedruckt werden muss. Dazu wird von fiskaltrust die ftCashBoxIdentification verwendet, Base64-kodiert, Sonderzeichen entfernt  und von der fiskaltrust.Middleware im Signatur-Block in der Response (ftSignatureType) zurückgeliefert. Die ftCashBoxIdentification selbst ist die frei wählbare, **eindeutige** Bezeichnung für eine Queue. Sie wird auch von der SCU als ClientId für die TSE verwendet. Deshalb ist es wichtig hierbei einen "printable string" mit maximal 20 Zeichen einzugeben. 
```

### Verwendung der oben beschriebenen Begriffe im Kontext von fiskaltrust

> Bei fiskaltrust wird im Sinne der Vereinfachung für die PC-Registrierkasse, Elektronische Registrierkasse und Eingabestation der Begriff "Kasse" verwendet. Diese Kasse entspricht der zu konfigurierenden "Queue" im Portal, welche zugleich die zu fiskalisierende und dem Finanzamt zu meldende Einheit darstellt.

Reviewer: ```suggestion
Bei fiskaltrust wird im Sinne der Vereinfachung für "Elektronische Aufzeichnungssysteme", "PC-Kasse", "Elektronische Registrierkasse" oder "Kasse" der Begriff "POS-System" verwendet. Dieses POS-System entspricht der zu konfigurierenden "Queue" im Portal, welche zugleich die zu fiskalisierende und dem Finanzamt zu meldende Einheit darstellt. Für die "Eingabestation" wird auch der begriff "Terminal" verwendet.
```

> Ein Kassensystem ("POS-System") lässt sich über eine oder mehrere Queues, sowie einen oder mehrere Konfigurationscontainer ("CashBoxen"), flexibel abbilden. 

Reviewer: ```suggestion
Eine Kasse ("POS-System") lässt sich über eine oder mehrere Queues, sowie einen oder mehrere Konfigurationscontainer ("CashBoxen"), flexibel abbilden. 
```

## CashBox

> Die CashBox ist ein Konfigurationscontainer, der die Konfiguration der einzelenen Komponenten der fiskaltrust.Middleware beinhaltet. Über die CashBox können die Konfigurationen miteinander verbunden werden. So kann die CashBox die Konfiguration von Queue, SCU und diversen Helper beinhalten und diese miteinander verbinden. 

Reviewer: ```suggestion
Die CashBox ist ein Konfigurationscontainer, der die Konfiguration der einzelenen Komponenten der fiskaltrust.Middleware beinhaltet. So kann die CashBox die Konfiguration von Queues, SCUs und diversen Helpern beinhalten und diese miteinander verbinden. 
```
Ich verstehe das nicht: "Über die CashBox können die Konfigurationen miteinander verbunden werden."

Integrität im Zusammenhang mit Daten bedeutet, dass diese seit der Erstellung unverändert geblieben sind. In Zusammenhang mit der TSE bedeutet es auch, dass keine Datensätze nachträglich entfernt wurden.

## KassenBetreiber ("PosOperator")

> Jeder, der ein Geschäft betreibt und dafür eine Registrierkasse benutzt ist für fiskaltrust ein KassenBetreiber. Wenn alles reibungslos läuft, bekommen KassenBetreiber von den erbrachten Dienstleistungen von fiskaltrust im Hintergrund nichts mit. Lediglich auf Anfrage von Steuerbehörden oder Wirtschaftsprüfern muss der KassenBetreiber auf die Einhaltung der KassenSichV hinweisen und mit Hilfe des KassenHändlers die Daten bereitstellen, die den Anforderungen der Steuervorschriften entsprechen.

Reviewer: ```suggestion
Jeder, der ein Geschäft betreibt und dafür eine POS-System benutzt ist für fiskaltrust ein KassenBetreiber. Wenn alles reibungslos läuft, bekommen KassenBetreiber von den erbrachten Dienstleistungen von fiskaltrust im Hintergrund nichts mit. Lediglich z.B. auf Anfrage von Steuerbehörden oder Wirtschaftsprüfern muss der KassenBetreiber auf die Einhaltung der KassenSichV hinweisen und mit Hilfe des KassenHändlers die Daten bereitstellen, die den Anforderungen der Steuervorschriften entsprechen.
```

> Der KassenBetreiber betreibt die fiskaltrust.Middleware lokal an seinem Standort.

Reviewer: ```suggestion
Der KassenBetreiber betreibt die fiskaltrust.Middleware lokal an seinem Standort oder in einem unter seiner Kontrolle stehenden Rechenzentrum.
```

## KassenHändler ("PosDealer")

> Der KassenHändler (kurz „Händler“) ist unabhängiger Vertriebshändler, der [Hard- und] Software für Kassen- und Abrechnungssysteme (kurz „Kassensysteme“) an den KassenBetreiber (kurz „Betreiber“) vertreibt.

Reviewer: ```suggestion
Der KassenHändler ist unabhängiger Vertriebshändler, der [Hard- und] Software für POS-Systeme an den KassenBetreiber vertreibt.
```

> Der Händler ist ein Systemintegrator, welcher POS Software in POS Hardware integriert und Dienstleistungen für die Installation und den Betrieb von Registrierkassen anbietet.  

Reviewer: ```suggestion
Der KassenHändler ist ein Systemintegrator, welcher POS-Software in POS-Hardware integriert und Dienstleistungen für die Installation und den Betrieb von POS-Systemen anbietet.  
```

> Händler verkaufen verschiedene Pos-Systeme für verschiedene Branchen weiter und stehen in direkter Beziehung zu PosOperatoren ("Betreiber"). Händler verfügen über hohe Fähigkeiten in Vertrieb, Projektmanagement und Kundenservice.

Reviewer: ```suggestion
KassenHändler verkaufen mitunter verschiedene POS-Systeme für verschiedene Branchen und stehen in direkter Beziehung zu PosOperatoren ("Betreiber"). KassenHändler verfügen meist über hohe Fähigkeiten in Vertrieb, Projektmanagement und Kundenservice.
```

> Der KassenHändler vertreibt die fiskaltrust.Middleware als solche oder als Bestandteil seiner Produkte bzw. Leistungen und übernimmt dafür den Support bei seinen Kunden.

Reviewer: ```suggestion
Der KassenHändler verbreitet die fiskaltrust.Middleware als solche oder als Bestandteil seiner Produkte und übernimmt dafür den Support bei seinen Kunden (KassenBetreibern).
```

## KassenHersteller ("PosCreator")

> Der KassenHersteller/PosCreator (kurz „Hersteller“) entwickelt, produziert bzw. programmiert Software für Kassen ( kurz „Kassensoftware“).

Reviewer: ```suggestion
Der KassenHersteller ("PosCreator") entwickelt, produziert bzw. programmiert Software für POS-Systeme.
```

> Er integriert das fiskaltrust.Ipos Interface in seine Software für Pos-Systeme. Er verfügt über hohe technische Fähigkeiten und vertreibt seine Produkte meist über KassenHändler. Berührungspunkte mit KassenBetreiber entstehen lediglich über spezielle Projekte oder über seine KassenHändler.

Reviewer: ```suggestion
Er integriert das fiskaltrust.Ipos-Interface in seine Software für POS-Systeme. Er verfügt meist über hohe technische Fähigkeiten und vertreibt seine Produkte oft über KassenHändler. Berührungspunkte mit KassenBetreiber entstehen in diesem Fall über spezielle Projekte oder über seine KassenHändler.
Zum Teil vertreiben KassenHersteller ihre POS-Systeme direkt - ohne zwischengeschaltetem KassenHändler - an KassenBetreiber. Dann füllt dieser KassenHersteller auch die Rolle des KassenHändlers aus.
```

> Der KassenHersteller integriert die fiskaltrust.Middleware in seine Software.

Reviewer: ```suggestion
Der KassenHersteller integriert die fiskaltrust.Middleware in seine POS-Systeme.
```

> Standorte eines Unternehmens sind die Orte (Plätze, abgegrenzte Räume), denen betriebliches Leistungspotential zur Durchführung der Leistungsprozesse zugeordnet ist (z.B. Event-Locations im Rahmen der Leistungserfüllung im Gastro/Catering-Bereich), sowie geographische Orte einer Betriebsstätte (z. B. Produktions-, Vertriebs-, Verwaltungsstätte).

Reviewer: ```suggestion
Standorte eines Unternehmens sind die Orte (Plätze, abgegrenzte Räume), denen betriebliches Leistungspotential zur Durchführung der Leistungsprozesse zugeordnet ist (z.B. Event-Locations im Rahmen der Leistungserfüllung im Gastro/Catering-Bereich), sowie geographische Orte einer Betriebsstätte (z.B. Produktions-, Vertriebs-, Verwaltungsstätte).
```

> In Deutschland muss jeder PosOperator für jedes POS-System und jeden Standort (Outlet) mindestens eine Queue betreiben.

Reviewer: ```suggestion
In Deutschland muss jeder KassenBetreiber für jedes POS-System und jeden Standort (Outlet) mindestens eine fiskaltrust.Queue betreiben.
```
